CITATION.bib

```diff
@@ -0,0 +1,6 @@
+@inproceedings{felten_toolkit_2023,
+	author = {Felten, Florian and Alegre, Lucas Nunes and Now{\'e}, Ann and Bazzan, Ana L. C. and Talbi, El Ghazali and Danoy, Gr{\'e}goire and Silva, Bruno Castro da},
+	title = {A {Toolkit} for {Reliable} {Benchmarking} and {Research} in {Multi}-{Objective} {Reinforcement} {Learning}},
```

Comment:
The trick of putting words inside {} is only for abbreviations and nouns that should never be not capitalized, e.g., RL, Gaussian. This could make the title capitalized even in a template where the titles are lowercase.